modules/commit-parser.js

```diff
@@ -1,5 +1,5 @@
 const git = require('./git.js')
-const REGEX_CONVENTIONAL_COMMIT_FORMAT = /(?<type>^[a-z\d]+)\(?(?<scopes>[a-z\d,\-]+)?\)?(?<breaking>!)?(?<colon>:{1})(?<space> {1})(?<body>.*)/
+const REGEX_CONVENTIONAL_COMMIT_FORMAT = /(?<type>^[a-z\d]+)\(?(?<scopes>[a-z_\d,\-]+)?\)?(?<breaking>!)?(?<colon>:{1})(?<space> {1})(?<body>.*)/
```

Comment:
change to have the same value as in the GH action:
https://github.com/ohpensource/ensure-conventional-commits-gh-action/blob/4d6143cc0449c6f99795567f49565d1ac98531b1/modules/constants.js#L37-L38

![image](https://github.com/ohpensource/generate-version-and-release-notes-gh-action/assets/97619732/6d917682-86ea-47db-b118-bb45847caaf4)

Reply:
now it accepts scopes with "_"